### ISSUE TITLE
snf-2011fe model

### ISSUE THREAD

**Issue by stevet40** [sncosmo/sncosmo#395]
I can't seem to load this source because the query to the snfactory website fails. This is probably an issue on their end, but if the data is available elsewhere somewhere stable it could be worth pointing to that instead.

Cc @aimalz @gnarayan @mattgrayling @astrofoley

**Comment by benjaminrose** [comment]
Yes. This is a known issue and I am in contact with snfactory to resolve the issue.

**Comment by stevet40** [comment]
Ah that's great! Thanks Ben!
